config/initializers/config.rb

```diff
@@ -0,0 +1,36 @@
+Config.setup do |config|
+  # Name of the constant exposing loaded settings
+  config.const_name = 'Settings'
```

Comment:
i guess here you define the const... so you should be able to do something like `Settings.some_setting` i guess